code/modules/mining/equipment_locker.dm

```diff
@@ -845,13 +845,15 @@
 			to_chat(user, "<span class='info'>[src] is only effective on lesser beings.</span>")
 			return
 
-/obj/item/lazarus_injector/emag_act()
+/obj/item/lazarus_injector/emag_act(mob/user)
 	if(!malfunctioning)
 		malfunctioning = 1
+		to_chat(user, "<span class='notice'>You override [src]'s safety protocols.</span>")
 
-/obj/item/lazarus_injector/emp_act()
+/obj/item/lazarus_injector/emp_act(mob/user)
 	if(!malfunctioning)
 		malfunctioning = 1
+		to_chat(user, "<span class='notice'>You override [src]'s safety protocols.</span>")
```

Comment:
can just use usr here instead of user. Don't have to give it in the parameters since usr is always available.

Reply:
Should always use `mob/user` if you can - it's more consistent with the rest of the codebase